sdk/loadtestservice/load-testing-rest/src/models.ts

```diff
@@ -260,21 +318,102 @@ export interface TestRunStatisticsModel {
+  /** File info */
+  testScriptFileInfo?: FileInfo;
+  /** File info */
+  userPropFileInfo?: FileInfo;
+  /** File info */
+  inputArtifactsZipFileInfo?: FileInfo;
+  /** Additional supported files for the test run */
+  additionalFileInfo?: Array<FileInfo>;
+}
+
+/** The output artifacts for the test run. */
+export interface TestRunOutputArtifacts {
+  /** File info */
+  resultFileInfo?: FileInfo;
+  /** File info */
+  logsFileInfo?: FileInfo;
+}
+
+/** Filters to fetch the set of metric */
+export interface MetricRequestPayload {
+  /** Get metrics for specific dimension values. Example: Metric contains dimension like SamplerName, Error. To retrieve all the time series data where SamplerName is equals to HTTPRequest1 or HTTPRequest2, the DimensionFilter value will be {"SamplerName", ["HTTPRequest1", "HTTPRequest2"} */
+  filters?: Array<DimensionFilter>;
+}
+
+/** Dimension name and values to filter */
+export interface DimensionFilter {
+  /** The dimension name */
+  name?: string;
+  /** The dimension values. Maximum values can be 20. */
+  values?: Array<string>;
+}
+
+/** Test run app component */
+export interface TestRunAppComponents {
+  /** Azure resource collection { resource id (fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}) : resource object } */
+  components: Record<string, AppComponent>;
+  /** Test run identifier */
+  testRunId?: string;
+  /** The creation datetime(ISO 8601 literal format). */
+  createdDateTime?: Date | string;
+  /** The user that created. */
+  createdBy?: string;
+  /** The last Modified datetime(ISO 8601 literal format). */
+  lastModifiedDateTime?: Date | string;
+  /** The user that last modified. */
+  lastModifiedBy?: string;
+}
+
+/** Test run server metrics configuration */
+export interface TestRunServerMetricConfig {
+  /** Test run identifier */
+  testRunId?: string;
+  /** Azure resource metrics collection {metric id : metrics object} (Refer : https://docs.microsoft.com/en-us/rest/api/monitor/metric-definitions/list#metricdefinition for metric id). */
+  metrics?: Record<string, ResourceMetric>;
+  /** The creation datetime(ISO 8601 literal format). */
+  createdDateTime?: Date | string;
+  /** The user that created. */
+  createdBy?: string;
+  /** The last Modified datetime(ISO 8601 literal format). */
+  lastModifiedDateTime?: Date | string;
+  /** The user that last modified. */
+  lastModifiedBy?: string;
+}
+
+/**
+ * Describes a poller for NotificationHubJob types.
+ */
+export type FileUploadAndValidatePoller = SimplePollerLike<
+  OperationState<TestGetFile200Response>,
+  TestGetFile200Response
+>;
+
+export type TestRunStatusPoller = SimplePollerLike<
+  OperationState<TestRunGet200Response>,
+  TestRunGet200Response
+>;
+
+export interface PolledOperationOptions extends OperationOptions {
+  /**
+   * Time delay between poll requests, in milliseconds.
+   */
+  updateIntervalInMs?: number;
```

Comment:
`OperationOptions` is a core-client concept that applies to high-level clients only. Rest-level clients such as this one don't have operation options. See [[here]](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/agrifood/agrifood-farming-rest/src/pollingHelper.ts) for an example.
```suggestion
export interface PolledOperationOptions {
  /**
   * Time delay between poll requests, in milliseconds.
   */
  updateIntervalInMs?: number;
```

Reply:
Thanks for sharing this, I have removed  OperationOptions from here.